Formula/wxpython.rb

```diff
@@ -60,11 +60,15 @@ def install
       # OpenGL and stuff
       "BUILD_GLCANVAS=1",
       "BUILD_GIZMOS=1",
-      "BUILD_STC=1"
+      "BUILD_STC=1",
     ]
 
     cd "wxPython" do
       system "python", "setup.py", "install", "--prefix=#{prefix}", *args
     end
   end
+
+  test do
+    system "python -c 'import wx ; print wx.version()'"
```

Comment:
Is it acceptable to have it as one line? e.g. 

`assert_match version.to_s, shell_output("python -c 'import wx ; print wx.version()'")`